spec/typed_parameters/comparison_spec.rb

```diff
@@ -25,10 +25,25 @@
       end
 
       context "when the request body does not adhere to the rules format" do
-        let(:name) { 1_000 }
+        context "when a non-specified parameter is included in the request body" do
+          let(:params) { { name: "M@", email: "matt@zipmark.com" } }
```

Comment:
@kurko spoke with Jake about this: https://zipmark.slack.com/archives/C32CZ771B/p1490814011401531